go/vt/vttablet/tabletserver/vstreamer/vstreamer.go

```diff
@@ -394,7 +391,8 @@ func (vs *vstreamer) parseEvents(ctx context.Context, events <-chan mysql.Binlog
 		case <-ctx.Done():
 			return nil
 		case <-hbTimer.C:
-			if err := injectHeartbeat(false, ""); err != nil {
+			checkResult, ok := vs.vse.throttlerClient.ThrottleCheckOKOrWaitAppName(ctx, vs.throttlerApp)
```

Comment:
Could you please explain what this throttler check is for?

Reply:
In that case I encourage you to use `vs.vse.throttlerClient.ThrottleCheckOK` as opposed to `vs.vse.throttlerClient.ThrottleCheckOKOrWaitAppName`. You're not interested in the extra sleep.

This makes sense now, thank you for clarifying.

Reply:
Ah, I see. Thank you! 🙇 That makes total sense — no need for the sleep here as it's not done in a tight loop. I should have at least checked the function comments. 🙂 I've made this change.